NewStarterProject/Controllers/UserController.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.AspNetCore.Mvc;
+using NewStarterProject.Dtos;
+using NewStarterProject.Model;
+using NewStarterProject.Services;
+
+namespace NewStarterProject.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class UserController : ControllerBase
+    {
+        private readonly UserService _userService;
+
+        public UserController(StarterProjectContext context)
+        {
+            _userService = new UserService(context);
```

Comment:
In Paxton10 we like to use Dependency Injection. 

For simplicity of the starter project I agree we could spin up a new instance of the UserService but for practice it might be worth considering if we can instead inject the UserService into the controller's constructure and use dependency injection instead?